docs/source/tutorials/resnet/index.md

```diff
@@ -227,7 +313,7 @@ from torchvision.models import resnet18
 from torchvision.transforms.v2 import Compose, Normalize, Resize, ToDtype, ToImage
 
 # Suppose you have the dataset in the datasets/ directory
-dataset_path = Path("datasets") / "human detection dataset"
+dataset_path = Path("datasets") / "human-detection-dataset-fine-tuning"
```

Comment:
When I ran fine-tuning I didn't get any feedback regarding training or validation accuracy:
<img width="858" alt="Screenshot 2025-05-21 at 10 32 27" src="https://github.com/user-attachments/assets/7cb2e149-260c-4de2-b932-d48b86845e7a" />

I think this should be included somewhere. (Ran it on google colab).